meshpy/header_functions.py

```diff
@@ -376,6 +377,8 @@ def set_header_static(
         Time increment per step.
     n_steps: int
         Number of time steps.
+    total_time: float
+        Total time of simulation
```

Comment:
Here the optional requirement of only two parameters could be added to the docstring?